packages/bpk-component-rating/src/BpkRating.js

```diff
@@ -48,84 +43,62 @@ const getMaxValue = (ratingScale) => {
   }
 };
 
-const getMediumRatingThreshold = (ratingScale) =>
-  getMaxValue(ratingScale) * MEDIUM_RATING_THRESHOLD;
-
-const getHighRatingThreshold = (ratingScale) =>
-  getMaxValue(ratingScale) * HIGH_RATING_THRESHOLD;
-
 export type Props = {
   ariaLabel: string,
+  className: ?string,
   ratingScale: $Values<typeof RATING_SCALES>,
+  showMaxValue: ?boolean,
   size: $Values<typeof RATING_SIZES>,
+  subtitle: ?string,
   title: string,
-  type: $Values<typeof RATING_TYPES>,
+  titleClassName: ?string,
   value: number,
-  vertical: boolean,
-  className: ?string,
-  subtitle: ?string,
 };
 
 const BpkRating = (props: Props) => {
   const {
     ariaLabel,
     className,
     ratingScale,
+    showMaxValue,
     size,
     subtitle,
     title,
-    type,
+    titleClassName,
     value,
-    vertical,
     ...rest
   } = props;
+
   const classNames = [getClassName('bpk-rating', className)];
-  const scoreStyles = [
-    getClassName(
-      'bpk-rating__component',
-      `bpk-rating--${size}-rating`,
-      type === RATING_TYPES.pill && `bpk-rating--${size}-pill`,
-    ),
+  const scoreStyles = [getClassName('bpk-rating__score')];
+  const textWrapperStyles = [
+    getClassName('bpk-rating__text-wrapper', titleClassName),
```

Comment:
Obviously this is not a good design, if we need to add this additional `className` to ensure alignment when passing in an image, but the consumer forgets this parameter, it will lead to an undesired UI displaying, so we need to delete this param but can resolve the alignment issue

Reply:
thanks, have pruned titleClassName